calabash/Classes/FranklyServer/Routes/LPBackdoorRoute.m

```diff
@@ -29,40 +34,41 @@ - (NSDictionary *) JSONResponseForMethod:(NSString *) method URI:(NSString *) pa
     selectorName = [selectorName stringByAppendingString:@":"];
   }
 
+  id argument = [data objectForKey:@"arg"];
+  if (!argument) {
+    LPLogError(@"Expected data dictionary to contain an 'arg' key");
```

Comment:
If you pass an empty array to:

```
LPInvoker invokeSelector:withTarget:arguments:
```

_and_ the selector takes no argument, all will be well.

The argument count must match the number of arguments in the selector.

LPInvoker doesn't handle `[NSNull null]` as nil.